packages/pwa-devdocs/src/venia-pwa-concept/design/global-style/index.md

```diff
@@ -0,0 +1,151 @@
+***Note:** If you are developing Venia themed components, you must install this font locally because the Muli font is currently not served by the backend server.*
+
+If the browser cannot find or use this font, it defaults to the following OS specific fonts:
+
+* **San Francisco** for Apple users
+* **Roboto** for Android users
+* **Segoe** for Windows users
+
+### Size specifications
+
+The Venia theme uses a root font size of **16px**.
+All other elements, except images and media queries, use this base value to calculate their relative sizes using `rem`.
+
+Examples using **16px** as the root font size:
+
+| `rem` size values        | Calculated size        |
+| ------------------------ | ---------------------- |
+| `padding: 1rem;`         | `padding: 16px;`       |
+| `border-radius: 1.5rem;` | `border-radius: 24px;` |
+| `font-size: 0.75rem;`    | `font-size: 12px;`     |
+{:style="table-layout:auto"}
+
+### Font weights
+
+The Venia theme uses a limited set of font weights in its design.
+For optimal component performance, use no more than 3 font weight variations.
+
+![Muli font weights][]{:width="350px"}
+
+Venia uses the following weights:
+
+| Common weight name | Value | Usage                                  |
+| ------------------ | ----- | -------------------------------------- |
+| Regular            | 400   | Regular text content                   |
+| Semi-bold          | 600   | Highlight important pieces of text     |
+| Bold               | 700   | Use only for the large text in banners |
+{:style="table-layout:auto"}
+
+## Layout
+
+### Grids
+
+Grid layouts should be in **multiples of 4** where possible, and
+the gap, or _gutter_, between each grid cell should be in **multiples of `8rem`**.
+
+### Borders
+
+Use masks and borders instead of drop shadows to separate different sections and visual components.
+
+![Masks and borders][]{:width="450px"}
+
+
+Use the following values for borders:
+
+| Property        | Value | Description                                 |
+| --------------- | ----- | ------------------------------------------- |
+| `border-radius` | `2px` | The roundness of an element's corner border |
+| `border-width`  | `1px` | The width of an element's border            |
+{:style="table-layout:auto"}
+
+{: .bs-callout .bs-callout-info}
+***Note:** For buttons, the `border-radius` is half the height of the button.
+For example, a button with a height of `3rem` should have a `border-radius` of `1.5rem`.*
+
+### Layers
+
+Regarding layers, do not use the Z-Index to re-order items within a layer unless it is absolutely necessary.
+
+## Animations
+
+Use the following guidelines to make component animations feel efficient and quick.
+
+| Animation property                      | Value                                   |
+| --------------------------------------- | --------------------------------------- |
+| Speed                                   | 8-16 milliseconds or 1 frame per second |
```

Comment:
I'm confused by this line. `16ms` does correspond to roughly `1 fps`, but it's not a value that people should use. Maybe suggest in the paragraph above that people should use increments of `16ms`?